event-broker test fails

Running `npm test` in `/packages/fxa-event-broker`

```
  44 passing (1s)
  1 failing

  1) Firestore database
       gets updated when the database changes:

      AssertionError: expected {} to deeply equal { test: 'testUrl' }
      + expected - actual

      -{}
      +{
      +  "test": "testUrl"
      +}
      
      at /Users/dcoates/code/fxa/packages/fxa-event-broker/test/lib/db/firestore.spec.ts:83:13
      at Generator.next (<anonymous>)
      at fulfilled (test/lib/db/firestore.spec.ts:8:58)
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1327)
┆Issue Number: FXA-1327

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -86,7 +86,7 @@ jobs:
       SKIP_DOCKER: true
     steps:
       - base-install
-      - run: ./packages/fxa-content-server/scripts/test-ci.sh
+      - run: ./.circleci/test-package.sh fxa-content-server
       - store_artifacts:
           path: ~/.pm2/logs
           destination: logs

--- a/packages/fxa-event-broker/scripts/test-ci.sh
+++ b/packages/fxa-event-broker/scripts/test-ci.sh
@@ -1,8 +1,0 @@
-#!/bin/bash -ex
-
-DIR=$(dirname "$0")
-
-cd $DIR/..
-npm ci
-# TODO enable when tests pass
-#npm test

--- a/packages/fxa-event-broker/src/test/lib/db/firestore.spec.ts
+++ b/packages/fxa-event-broker/src/test/lib/db/firestore.spec.ts
@@ -66,20 +66,24 @@ describe('Firestore database', () => {
 
   it('gets updated when the database changes', async () => {
     const data: ClientWebhooks = {};
-    const stop = db.listenForClientIdWebhooks(
-      (changed, removed) => {
-        Object.assign(data, changed);
-        for (const key of Object.keys(removed)) {
-          delete data[key];
+    const change = new Promise<() => void>((resolve, reject) => {
+      const done = db.listenForClientIdWebhooks(
+        (changed, removed) => {
+          Object.assign(data, changed);
+          for (const key of Object.keys(removed)) {
+            delete data[key];
+          }
+          resolve(done);
+        },
+        err => {
+          reject(err);
         }
-      },
-      err => {
-        cassert.fail();
-      }
-    );
+      );
+    });
     // Manually insert into the db
     const document = (db as any).db.doc('fxatest-clients/test');
     await document.set({ webhookUrl: 'testUrl' });
+    const stop = await change;
     cassert.deepEqual(data, { test: 'testUrl' });
 
     // Manually delete from the db

